packages/lib/slots.ts

```diff
@@ -132,7 +125,7 @@ function buildSlotsWithDateRanges({
         };
       }
 
-      slots.set(slotData.time.toISOString(), slotData);
```

Comment:
Key is now valueOf(), as this is a much faster key to determine, toISOString() is much slower.